### ISSUE TITLE
Server Disconnection Issue

### ISSUE THREAD

**Issue by RalseiSmolToo** [dankmeme01/globed2#120]
experiencing an annoying disconnection dialogue after a split second of running it on background/not on the main activity

Platform: Android32(probably Android64 has it too)

Issue Occured: Since the Android Port came

may add an image about this issue later

**Comment by dankmeme01** [comment]
is this still happening in v1.3.1?

**Comment by AverageMaipuperson** [comment]
it's also happening to me

**Comment by RalseiSmolToo** [comment]
> is this still happening in v1.3.1?

yeah, but not rarely after the bugfix update. It shows me a wierd disconnection dialogue(sorry I didn't captured it, but next time for sure if it happens again)

**Comment by dankmeme01** [comment]
closing as stale, reopen if happens in v2.0.0
